overrides/ios-override.json

```diff
@@ -1,5 +1,12 @@
 {
     "features": {
+        "adAttributionReporting": {
+            "state": "disabled",
```

Comment:
Are we intending to leave this disabled?

Reply:
Yes, disabled until we will be running an ASA campaign and would like to have reporting for it.